tests/common/fixtures/populate_fdb.py

```diff
@@ -65,10 +65,14 @@ def __prepareVlanConfigData(self):
                     "vlan": vlan,
                     "index": mgFacts["minigraph_port_indices"][port]
                 })
+        vlan_interfaces = {}
+        for vlan in mgFacts["minigraph_vlan_interfaces"]:
+            if isinstance(ipaddress.IPNetwork(vlan['addr']), ipaddress.IPv4Network):
```

Comment:
The IPNetwork object should have a `version` property. It may be better to use the `version` property to tell whether it is IPv4 or IPv6.

Reply:
Thanks. 
Updated